client/src/main/java/org/hyperskill/database/clientside/Commands.java

```diff
@@ -0,0 +1,14 @@
+package org.hyperskill.database.clientside;
+
+import com.beust.jcommander.IParameterValidator;
+import com.beust.jcommander.ParameterException;
+
+public class Commands implements IParameterValidator {
+    public void validate(String name, String value)
+            throws ParameterException {
+        value = value.trim().toLowerCase();
+        if (!(value.equals("set") || value.equals("get") || value.equals("delete"))) {
```

Comment:
That's fine, except for potential NPE, which I described in `Client` class
But could be simplified by creating a `Set` of allowed commands and checking if passed value is in this set
The set could also be used in exception message to create a string of allowed commands `allowed.stream().collect(Collectors.joining("/"))`